Makefile

```diff
@@ -311,20 +253,22 @@ $(SSL_PKGCONFNAME): valkey_ssl.h
 	@echo Libs.private: -lssl -lcrypto >> $@
 
 install: $(DYLIBNAME) $(STLIBNAME) $(PKGCONFNAME) $(SSL_INSTALL)
```

Comment:
I wonder if there is a good way to create helper functions in a Makefile for these install/pkgconfig targets?
We will have for of them when cluster is incorporated, but maybe being explicit is the best option..

Reply:
Not sure, will look into it.